api/v1alpha1/zero_trust_workload_identity_manager_types.go

```diff
@@ -62,24 +62,32 @@ type ZeroTrustWorkloadIdentityManagerList struct {
 
 // ZeroTrustWorkloadIdentityManagerSpec defines the desired state of ZeroTrustWorkloadIdentityManager
 type ZeroTrustWorkloadIdentityManagerSpec struct {
-	// logLevel supports value range as per [kubernetes logging guidelines](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md#what-method-to-use).
-	// +kubebuilder:default:=1
-	// +kubebuilder:validation:Minimum:=1
-	// +kubebuilder:validation:Maximum:=5
-	// +kubebuilder:validation:Optional
-	LogLevel int32 `json:"logLevel,omitempty"`
-
 	// namespace to install the deployments and other resources managed by
 	// zero-trust-workload-identity-manager.
 	// +kubebuilder:validation:Optional
 	// +kubebuilder:default:="zero-trust-workload-identity-manager"
 	Namespace string `json:"namespace,omitempty"`
 
+	// logLevel sets the logging level for the operand.
+	// Valid values are: debug, info, warn, error.
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:validation:Enum=debug;info;warn;error
+	// +kubebuilder:default:="warn"
+	LogLevel string `json:"logLevel,omitempty"`
+
+	// logFormat sets the logging format for the operand.
+	// Valid values are: text, json.
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:validation:Enum=text;json
+	// +kubebuilder:default:="text"
+	LogFormat string `json:"logFormat,omitempty"`
```

Comment:
Seemingly the log format config entry is defined here but never got used in `cmd/zero-trust-workload-identity-manager/main.go`.